app/workers/import_worker.rb

```diff
@@ -35,6 +37,14 @@ def import_rows
     CSV.new(import_contents).reject(&:blank?)
   end
 
+  def process_mutes
+    import_rows.each do |row|
+      target_account = Account.find_local(row.first)
```

Comment:
I think this isn't quite right... Mutes don't apply to local users only. This could should be more or less equivalent with the block import. If the account isn't there, look it up. It's counterintuitive, because if it's a "bad" account, why add it to the system? But if we don't, and it gets added later by someone else, we don't have the block/mute and that's worse.